appengine/pubsub/app.standard.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: ruby25
+entrypoint: bundle exec ruby app.rb
+
```

Comment:
Does this need an instance class? https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/appengine/standard-hello_world/app.yaml#L18

Reply:
Sure, let's add F2 as in the other sample, It isn't necessary, but seems like a reasonable setting.
I'll test before committing the PR

Reply:
app deploys and runs with instance_class: F2